main.js

```diff
@@ -0,0 +1,30 @@
+var amountInput = document.querySelector('.amountInput');
+var amountEntered = function(event) {
+  event.target.value
```

Comment:
This is a good start, but this function isn't going to do much unless you return or assign that input value.